doc/install.rst

```diff
@@ -1,83 +1,88 @@
-:ref:`newbie-scipy-sec-label`.
+  $ pip install tulip
 
-The default synthesis tool for GR(1) specifications is `gr1c
-<http://scottman.net/2012/gr1c>`_. Please install at least version 0.9.0. If you
-do not already have it, the introduction of `the manual of gr1c
-<https://tulip-control.github.io/gr1c/>`_ is a good place to begin.
+or from source::
 
-The following are optional dependencies, listed with a summary of dependent
-features:
+  $ pip install .
 
-* `Matplotlib <http://matplotlib.org/>`_ -- many visualization features
+``pip`` installs Python dependencies `automatically
+<https://pip.pypa.io/en/stable/reference/pip_install/#installation-order>`_.
+They are listed in ``install_requires`` within ``setup.py``.
+The only Python packages that you may want to install yourself,
+in order to link them properly are:
 
-* `Graphviz <http://www.graphviz.org/>`_ -- generation of images (e.g., PNG
-  files) from dot code
+* `NumPy <http://numpy.org/>`_
+* `SciPy <http://www.scipy.org/>`_
 
-* `CVXOPT <http://cvxopt.org/>`_ -- construction and manipulation of discrete
-  abstractions
+The following are optional dependencies,
+listed with a summary of dependent features:
 
-For computing discrete abstractions from hybrid system descriptions, it is
-highly recommended---but not required---that you install `GLPK
-<http://www.gnu.org/s/glpk/>`_ (a fast linear programming solver). Note that you
-need to install GLPK *before* installing CVXOPT and follow the instructions in
-CVXOPT installation to ensure it recognizes GLPK as a solver. If you are a
-`MacPorts <http://www.macports.org/>`_ user, please note that MacPorts does not
-do this linking automatically.
+* `Matplotlib <http://matplotlib.org/>`_ --
+  many visualization features
 
-Once all of the above preparations are completed, you can install TuLiP::
+* `Graphviz <http://www.graphviz.org/>`_ --
+  to plot graphs, for example discrete state machines
 
-  $ pip install .
+* `CVXOPT <http://cvxopt.org/>`_ --
+  construction and manipulation of discrete abstractions
 
-TuLiP may instead be installed `from PyPI <https://pypi.python.org/pypi/tulip>`_::
+* `GLPK <http://www.gnu.org/s/glpk/>`_ --
+  fast linear programming solver
 
-  $ pip install tulip
+For computing discrete abstractions from hybrid system descriptions,
+it is highly recommended that you install both CVXOPT and GLPK.
+Note that you need to install GLPK *before* installing CVXOPT,
+and follow the `CVXOPT installation instructions
+<http://cvxopt.org/install/index.html>`_
+to link CVXOPT to GLPK.
+(If you use
+`MacPorts <http://www.macports.org/>`_,
+please note that MacPorts does not do this linking automatically.)
 
-The above commands include checking of dependencies and automatic installation
-of missing Python packages. (N.B., not all dependencies are Python packages.)
+The latest release of TuLiP can be downloaded also from
+`SourceForge
+<http://sourceforge.net/projects/tulip-control/files/>`_.
```

Comment:
When I try to visit <http://sourceforge.net/projects/tulip-control/files/>, the HTTP connection is upgraded to HTTPS automatically (thus becoming <https://sourceforge.net/projects/tulip-control/files/>). Is there any motivation to keep the scheme as http and not https here?